apps/article/stylesheets/amp.styl

```diff
@@ -0,0 +1,55 @@
+- amp-header-height = 54px
+
+.body-amp
+  max-width 580px
+
+  .amp-header
+    height amp-header-height
+    border-bottom 1px solid gray-lighter-color
+    display flex
+    align-items center
+    a
+      margin-left 20px
+
+  .article-header
+    margin-top 50px
+
+  amp-carousel
```

Comment:
Generally speaking, it's best to use flat namespaced classes. There're a couple other benefits to namespacing over scoped selectors that was [summed up well by Phillip Walton](https://philipwalton.com/articles/css-architecture/) and formalized into the [Block-Element-Modified, or BEM, method](http://getbem.com/). One way to clean this up is to use the stylus ampersand, e.g.

````stylus
.article-amp
  &-carousel
    // ...
  &-header
````